apps/nexus/app/components/docs/DocsComponentsGallery.css

```
@@ -0,0 +1,386 @@
+ * past the 3000-line mark where Vite's separate dev sub-request for a style
+ * block can resolve against a misaligned descriptor slice — reporting a
+ * PostCSS error for CSS that is valid on disk and dropping the page's
+ * styling entirely. `test/guards/sfc-size-budget.test.ts` enforces this.
+ *
+ * Unscoped as a consequence, which is safe here: every selector is under the
+ * `.docs-gallery` namespace and the block used no `:deep()`.
+ */
+
+.docs-gallery {
+  --docs-gallery-line: color-mix(in srgb, var(--docs-border) 72%, transparent);
+  border: 1px solid var(--docs-gallery-line);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)